lib/pdk/report.rb

```diff
@@ -85,6 +85,9 @@ def to_junit(target = self.class.default_target)
     # @param target [#write] an IO object that the report will be written to.
     #   Defaults to PDK::Report.default_target.
     def to_text(target = self.class.default_target)
+      # Extra defaulting here, b/c the Class.send method will pass in nil
+      target ||= self.class.default_target
```

Comment:
Good idea. And the formatting happens in the `to_text` method of the `Event` class.